guides/howto/custom_error_pages.md

```diff
@@ -167,9 +167,9 @@ However, implementing the `Plug.Exception` protocol by hand can be convenient in
 
 ## Actionable errors
 
-Exception actions are functions that can be triggered by the error page, and they're basically a list of maps defining a `label` and a `handler` to be executed.
+Exception actions are functions that can be triggered from the error page, and they're basically a list of maps defining a `label` and a `handler` to be executed. As an example, Phoenix will display an error if you have pending migrations and will provide a button on the error page to perform the pending migrations.
 
-They are rendered in the error page as a collection of buttons and follow the format of:
+When `debug_errors` is `true`, they are rendered in the error page as a collection of buttons and follow the format of:
```

Comment:
The user may have turned this off previously in the guide, so important to note.